manifests/m/Microsoft/PowerShell/7.3.6.0/Microsoft.PowerShell.locale.en-US.yaml

```diff
@@ -0,0 +1,83 @@
+# Created with Komac v1.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.3.6.0
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://github.com/PowerShell/PowerShell
+PublisherSupportUrl: https://github.com/PowerShell/PowerShell/issues
+Author: Microsoft Corporation
+PackageName: PowerShell
+PackageUrl: https://microsoft.com/PowerShell
+License: MIT
+LicenseUrl: https://github.com/PowerShell/PowerShell/blob/master/LICENSE.txt
+Copyright: Copyright (c) Microsoft Corporation
+CopyrightUrl: https://github.com/PowerShell/PowerShell/blob/master/LICENSE.txt
+ShortDescription: PowerShell is a cross-platform task automation solution made up of a command-line shell, a scripting language, and a configuration management framework. PowerShell runs on Windows, Linux, and macOS.
+Description: |-
+  PowerShell is a cross-platform (Windows, Linux, and macOS) automation and configuration tool/framework that works well with your existing tools and is optimized for dealing with structured data (e.g.
+  JSON, CSV, XML, etc.), REST APIs, and object models.
+  It includes a command-line shell, an associated scripting language and a framework for processing cmdlets.
+Moniker: pwsh
+Tags:
+- command-line
+- cross-platform
+- open-source
+- powershell
+- pwsh
+- shell
+ReleaseNotes: |-
+  Documentation and Help Content
+  - Update the link for getting started in README.md (#19947)
+  SHA256 Hashes of the release artifacts
+  - hashes.sha256
+  - 3ECBFCD55C66ECF3B7E04206
+  - powershell_7.3.6-1.deb_amd64.deb
+  - A2F17EABFF85841FBE6C9E57
+  - powershell-7.3.6-1.cm.x86_64.rpm
+  - DCAC341B13A34EC19EB6CF86
+  - powershell-7.3.6-1.rh.x86_64.rpm
+  - DDA0F189B43188FACDA6F1D4
+  - powershell-7.3.6-linux-alpine-x64.tar.gz
+  - 184DBEB80CF6C9057C0BFB5E
+  - powershell-7.3.6-linux-arm32.tar.gz
+  - 26E476F0F249DAB51A30797B
+  - powershell-7.3.6-linux-arm64.tar.gz
+  - 244CE4D22650BF4A4BD722C3
+  - powershell-7.3.6-linux-x64.tar.gz
+  - 96B5F03D0C3D56F9A2D8A5DC
+  - powershell-7.3.6-linux-x64-fxdependent.tar.gz
+  - F23E31A5BC3E65CA581BF9BE
+  - powershell-7.3.6-osx-arm64.pkg
+  - 515980B61C0F068A4915D1FD
+  - powershell-7.3.6-osx-arm64.tar.gz
+  - 0DA3B51D61ADFDB4E5ACB2AF
+  - powershell-7.3.6-osx-x64.pkg
+  - 5D62564767756527DE52D878
+  - powershell-7.3.6-osx-x64.tar.gz
+  - 6B04DC76C954CAEB25D39069
+  - PowerShell-7.3.6-win-arm32.zip
+  - 2ABA0B8905D8C9846F7B0457
+  - PowerShell-7.3.6-win-arm64.zip
+  - 5DDBA433BF68DB74F7D47026
+  - PowerShell-7.3.6-win-fxdependent.zip
+  - 031FC07CED5BC7CCDF5E6154
+  - PowerShell-7.3.6-win-fxdependentWinDesktop.zip
+  - D26C224D781BD3DB16E36F18
+  - PowerShell-7.3.6-win-x64.msi
+  - A22114C3B00D50CF7295D35E
+  - PowerShell-7.3.6-win-x64.zip
+  - 5D4D658D2F54B88F99BD01BA
+  - PowerShell-7.3.6-win-x86.msi
+  - 85252C77738D38CF28AA18EE
+  - PowerShell-7.3.6-win-x86.zip
+  - BFBB03711669CF047276D13D

```

Comment:
```suggestion
ReleaseNotes: |-

  Build and Packaging Improvements

  Bump .NET to 7.0.306
    - Update Notices file
    - Don't publish notice on failure because it prevents retry
    - Bump .NET to 7.0.306 (#19945)
    - Remove the property disabling optimization (#19952)
    - Add ProductCode in registry for MSI install (#19951)
    - Update variable used to bypass the blocking check for multiple NuGet feeds (#19953)
    - Change System.Security.AccessControl preview version to stable version (#19931)

  Documentation and Help Content
    - Update the link for getting started in README.md (#19947)
```